Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java

```diff
@@ -0,0 +1,144 @@
+package nl.soccar.gameserver.model.session;
+
+import nl.soccar.gameserver.controller.socnet.message.BallSyncMessage;
+import nl.soccar.gameserver.controller.socnet.message.MovePlayerMessage;
+import nl.soccar.gameserver.controller.socnet.message.PlayerStartedGameMessage;
+import nl.soccar.gameserver.controller.socnet.message.PlayerSyncMessage;
+import nl.soccar.gameserver.model.PlayerWrapper;
+import nl.soccar.gameserver.util.CarUtilities;
+import nl.soccar.gameserver.util.MapUtilities;
+import nl.soccar.library.Car;
+import nl.soccar.library.Game;
+import nl.soccar.library.GameSettings;
+import nl.soccar.library.Map;
+import nl.soccar.library.enumeration.GameStatus;
+import nl.soccar.library.enumeration.HandbrakeAction;
+import nl.soccar.library.enumeration.SteerAction;
+import nl.soccar.library.enumeration.ThrottleAction;
+import nl.soccar.physics.GameEngine;
+import nl.soccar.physics.models.BallPhysics;
+import nl.soccar.physics.models.CarPhysics;
+import nl.soccar.socnet.message.Message;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * @author Lesley
+ */
+public final class GameWrapper {
+
+    private final SessionWrapper session;
+    private final Game game;
+
+    private final GameEngine engine;
+    private final Timer timer;
+
+    public GameWrapper(SessionWrapper session, Game game) {
+        this.session = session;
+        this.game = game;
+
+        engine = new GameEngine(game);
+        timer = new Timer();
+    }
+
+    public void start() {
+        initializeWorldObjects();
+        sendGameInformation();
+
+        engine.start();
+        game.start();
+
+        timer.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                if (game.getStatus() == GameStatus.STOPPED) {
+                    stop();
+                    return;
+                }
+
+                List<Message> messages = getSynchronisationMessages();
+                session.getRoom().getPlayers().stream()
+                        .map(PlayerWrapper::getConnection)
+                        .forEach(c -> messages.forEach(c::send));
+            }
+        }, 0, 75);
+    }
+
+    public void stop() {
+        timer.cancel();
+
+        game.stop();
+        engine.stop();
+    }
+
+    public void movePlayer(PlayerWrapper player, SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)